rfcs/0104-rewrite-taskcluster-events.md

```diff
@@ -0,0 +1,30 @@
+# RFC 104 - rewrite taskcluster-events with a proper read-only RabbitMQ / websocket proxy
+* Comments: [#104](https://api.github.com/repos/taskcluster/taskcluster-rfcs/issues/104)
+* Proposed by: @jonasfj
+
+# Summary
+
+Rewrite taskcluster-events to support:
```

Comment:
I'm curious: which of these are parts of a new PulseListener/PulseConnection, and which are parts of the proposed events service?